Battle/Owl.cpp

```diff
@@ -80,12 +81,19 @@ void Owl::process() {
 
     if (position->x > BOMB_AREA_OFFSET && position->x < WINDOW_WIDTH - BOMB_AREA_OFFSET - position->w) {
         if (main_.gameplay().frame % 10 == 0) {
-            Bomb *b = new Bomb(main_.graphics->bombs, main_);
+            Bomb *b = NULL;
+            if (rand() % 5 == 1) {
+                b = new Egg(main_.graphics->bombs, main_);
+                b->damage = -Player::BOMBPOWERCLASSES[owner->bombpowerclass].damage;
```

Comment:
Misschien is dit een beetje raar... Nu is het aantal hp wat je erbij krijgt gebaseerd op hoe goed iemands bommen zijn... Is misschien logischer als dit een vaste waarde is?